packages/Text/src/index.native.js.flow

```diff
@@ -9,7 +9,7 @@ declare module "constelation-text" {
       color?: string,
       decorationLine?: 'none' | 'underline' | 'line-through' | 'underline line-through',
       fontFamily?: string,
-      height?: number,
+      height?: number | string,
```

Comment:
Oh you're mapping height to lineHeight? IMO that feels a little too custom since Text supports standard `width` & `height` props (including `minWidth` etc. with percentage support). I'd suggest supporting all the standard layout props plus a `lineHeight` prop.

I'll revert this change for now though!